src/assets/manifest.json

```
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Let's update this so it's specific to LibreWeather

```suggestion
  "short_name": "LibreWeather",
  "name": "LibreWeather Frontend",
```